### PR TITLE
3 filter todos by owner

### LINKED ISSUE #3: filter todos by owner
e2e test + jasmine unit tests

### PR DESCRIPTION
**Pull request by KenBroden**
- Owner is filtered on the server
- don't know what the todo cards look like yet
  - I don't think we will be able to see them until the server and mongodb stuff is fixed
  - might be a good reason to move up those issues/tasks
- very basic cypress tests are in
- Karma tests all pass, good coverage besides the method in the pic, we might need the other filters to get coverage on getTodoById() in todo.service.mock.ts
image

### CODE REVIEW

File: client/src/app/todos/todo-card.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@if (this.todo) {
+  <mat-card class="todo-card">
+    <mat-card-header>
+      <mat-card-title class="todo-card-name">{{ this.todo.owner }}</mat-card-title>
```
Comment (mahon479) [inline client/src/app/todos/todo-card.component.html]: should the class = "todo-card-owner"?

File: client/src/app/todos/todo-card.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <mat-card-subtitle class="todo-card-company">{{ this.todo.category }}</mat-card-subtitle>
```
Comment (mahon479) [inline client/src/app/todos/todo-card.component.html]: Should the class here equal "todo-card-category"?

**Review by mahon479 — COMMENTED** [review]
I added two comments for possible changes we may need to make.

**Comment by KenBroden** [comment]
ng -test kept passing in vscode in the lab, so I just re-ran the tests on github within the pull request.  They eventually passed

**Review by mahon479 — APPROVED** [review]
Looks good, ready to merge.

**Pull request merged**